frontend/src/lib/components/LemonTable/TableRow.tsx

```diff
@@ -32,10 +32,13 @@ function TableRowRaw<T extends Record<string, any>>({
             <tr data-row-key={rowKeyDetermined} {...onRow?.(record)} className={rowClassNameDetermined}>
                 {!!expandable && rowExpandable >= 0 && (
                     <td>
-                        {rowExpandable && (
+                        {!!rowExpandable && (
                             <LemonButton
                                 type={isRowExpanded ? 'highlighted' : 'stealth'}
-                                onClick={() => setIsRowExpanded((state) => !state)}
+                                onClick={() => {
+                                    setIsRowExpanded((state) => !state)
+                                    !isRowExpanded && expandable.onRowExpanded && expandable.onRowExpanded(record)
```

Comment:
at this point `isRowExpanded` was false when I hit expand, but I'm not sure exactly when `setIsRowExpanded` state change happens